src/rez/utils/pip.py

```diff
@@ -332,6 +332,22 @@ def is_pure_python_package(installed_dist):
     return (is_purelib == "true")
 
 
+def is_entry_points_scripts_package(installed_dist):
```

Comment:
`is_pure_python_package` has unit tests. It would be cool if `is_entry_points_scripts_package` would also have a unit test. See https://github.com/AcademySoftwareFoundation/rez/blob/master/src/rez/tests/test_pip_utils.py#L144.

Also, I'm tempted to say that `scripts` in the function name is not necessary since all entry points are scripts.

Reply:
Thanks, I'll add unit tests. There are other uses for entry points other than scripts, like defining plugins or hooks.